dj/models/table.py

```diff
@@ -53,21 +53,21 @@ class TableBase(BaseSQLModel):
     cost: float = 1.0
 
 
-class TableNodeRevision(BaseSQLModel, table=True):  # type: ignore
-    """
-    Link between a table and a node revision.
-    """
-
-    table_id: Optional[int] = Field(
-        default=None,
-        foreign_key="table.id",
-        primary_key=True,
-    )
-    node_revision_id: Optional[int] = Field(
-        default=None,
-        foreign_key="noderevision.id",
-        primary_key=True,
-    )
+# class TableNodeRevision(BaseSQLModel, table=True):  # type: ignore
```

Comment:
Just remove this?

Reply:
Done

Reply:
It's still here? 😛

Reply:
Thanks @betodealmeida!
![image](https://user-images.githubusercontent.com/43911210/224521072-3ca4817a-e9cc-4e21-ac80-c4144b0c49b0.png)

Reply:
Ok, now it's really really removed. lol